src/features/courses/course-controller.ts

```diff
@@ -25,7 +25,7 @@ import curriculumRepository from '../curriculum/curriculum-repository';
 import CurriculumUnitContent from '../../database/models/curriculum-unit-content';
 import CurriculumTopicContent from '../../database/models/curriculum-topic-content';
 import CurriculumWWTopicQuestion from '../../database/models/curriculum-ww-topic-question';
-import WebWorkDef, { Problem } from '@rederly/webwork-def-parser';
+import WebWorkDef, { Problem } from '@rederly/rederly-utils/lib/webwork-def-parser';
```

Comment:
It seems a bit unintuitive to have to pull this out from a subdirectory of an npm package.

Reply:
Yeah, I wanted to keep the exports the same for now, the default export itself is probably the most annoying part

In the future I might refactor